README.md

```diff
@@ -70,6 +70,9 @@
 
 - [Flutter Web and Flame](https://github.com/AseemWangoo/experiments_with_web) - Hands on with Flame for web. By [Aseem Wangoo](https://flatteredwithflutter.com/how-to-create-game-in-flutter-web-using-flame/)
 
+<!--lint ignore double-link-->
+- [Simple Platformer - Flame Game Development Series](https://youtube.com/playlist?list=PLiZZKL9HLmWPyd808sda2ydG-dhexNONV) - 2D platformer without using Forge2d. By [DevKage](https://www.youtube.com/devkage)
```

Comment:
I think you have to use a reference style link on the last link here to stop the linter from complaining (because the link is somewhere else in the file too).
https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
Oh, I didn't know about ref style links. Let me try that.